source/CardGames.py

```diff
@@ -433,12 +446,7 @@ def marshall_first_turn(self):
       self.display_board_general()
 
 game = GameSetup()
+game.ReadRules()
```

Comment:
Cool stuff. Could you call ReadRules() in the first line of the start_game() function instead of at the bottom just to keep as much as we can contained in start_game()?

Reply:
Thanks, Sean! I will be sure to move the call for ReadRules() to the first line of the start_game() function.